enterprise/ha/src/test/java/org/neo4j/ha/HowSensitiveIsClusteringToSlowLoggingInsideTheStateMachinesIT.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2002-2015 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.ha;
+
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+
+import org.neo4j.cluster.client.ClusterJoin;
+import org.neo4j.cluster.logging.AsyncLogging;
+import org.neo4j.kernel.impl.ha.ClusterManager.ManagedCluster;
+import org.neo4j.kernel.impl.util.Listener;
+import org.neo4j.test.ha.ClusterRule;
+
+@Ignore("This test is useful for manual runs, but it runs indefinitely and thus cannot be enabled")
+public class HowSensitiveIsClusteringToSlowLoggingInsideTheStateMachinesIT
```

Comment:
Should this be converted to some kind of tool or perf test instead. 
Do not believe anybody will ever run it manually, and people that will need something similar will never expect to find it inside one of the disabled tests